contents/handbook/people/team-structure/platform/objectives.mdx

```diff
@@ -1,11 +1,42 @@
-* **Proposed Objective**:
-    * PostHog Cloud in the US remains reliable and launch EU Cloud
-* **Proposed Key Results**:
-    * Multiple organizations using EU hosted PostHog Cloud without regressions in either EU or US cloud
-    * Reliable monitoring and alerting on all key services
-    * Upgrade to Kubernetes 1.25 to unblock key customers
-    * Meet SOC 2 compliance requirements for all cloud infrastructure
-* **Rationale**:
-    * To nail self-serve we need to make it as easy as possible for customers to sign up and get started
-    * For businesses in the EU today, the main barrier is needing to self host to keep data in the EU, this limits trust in our brand and the potential adoption we can drive
-    * Shipping a secure cloud in the EU will also encourage many businesses to get started with PostHog Cloud, even if they ultimately intend to self-host
\ No newline at end of file
+- Objective: High uptime and increase the resilience/redundancy
+    - Key results:
+        - Team to fill in
+    - Why?
+        - A lack of reliability will cause churn.
+- Objective: Meet the compliance needs of 5 companies that we cannot meet today
+    - Key result: EU Cloud is successfully managed
+    - Key result: SOC2 completed and security audit
```

Comment:
A couple of questions:
* SOC2 type 1 or SOC2 type2? 
* _completed_ defined as: we are certified or we are ready to be certified? Getting certified usually takes between 3 and 12 months so I'm not sure if we should commit to get certified by EOQ
* security audit: which kind of security audit?

Reply:
(Jumping in) This would be SOC 2 Type 1 as it's our first audit. Given the timeline, I would probably define it as 'All compliance tasks completed in Drata and audit booked' as we can't control the date the auditors are available and how long they take to write the report. 

(If we go ahead with this though, I'm happy to get in touch with them again and talk timelines, would just need to understand from this team how long their bits in Drata would take.)